generator/source/faq/grid.rst

```diff
@@ -2,46 +2,129 @@
 hide: true
 tags: [faq]
-title: Grid FAQ - Hyperledger Grid in General
+title: General FAQ
 permalink: /faq/grid/
 # Copyright (c) 2018-2019, Bitwise IO, Inc.
 # Copyright (c) 2018, Intel Corporation.
 # Licensed under Creative Commons Attribution 4.0 International License
 # https://creativecommons.org/licenses/by/4.0/
 ---
-Grid FAQ: Hyperledger Grid in General
-=====================================
-.. class:: mininav
 
-PREVIOUS_ FAQ_
+General Hyperledger Grid FAQ
+============================
 
 .. contents::
 
+.. _faq-what-is-hyperledger-grid:
 
-What is Grid?
--------------
+What is Hyperledger Grid?
+-------------------------
 
-Hyperledger Grid is a project for building supply chain solutions. It includes
-a set of libraries, data models, and SDKs to accelerate development for supply
-chain smart contracts and client interfaces.
+Hyperledger Grid is a platform for building supply chain solutions that include
+distributed ledger components. This project's goal is to speed up the development
+of modular blockchain-based solutions for cross-industry supply chain problems.
 
-What are some useful Grid links?
---------------------------------
+Hyperledger Grid provides a set of libraries, data models, and SDKs that can be
+used to develop supply chain smart contracts and client interfaces.
 
-GitHub repository for Grid
-    https://github.com/hyperledger/grid
-Grid documentation:
-    https://grid.hyperledger.org/docs/
-Grid chat main channel
-    https://chat.hyperledger.org/channel/grid
-Grid mailing list (not as popular as the chat channel above)
-    https://lists.hyperledger.org/g/grid
-Grid FAQ
-    https://grid.hyperledger.org/faq/
+For more information:
 
-.. class:: mininav
+* `About Hyperledger Grid </about>`__
 
-PREVIOUS_ FAQ_
+* `Hyperledger Grid project overview <https://www.hyperledger.org/projects/grid>`__
+  at hyperledger.org
+
+* `Hyperledger Grid wiki <https://wiki.hyperledger.org/display/grid>`__
+  at hyperledger.org
+
+* `Announcing Hyperledger Grid
+  <https://www.hyperledger.org/blog/2019/01/22/announcing-hyperledger-grid-a-new-project-to-help-build-and-deliver-supply-chain-solutions>`__
+  (Hyperledger blog)
+
+.. _faq-what-is-the-status-of-this-project:
+
+What is the status of this project?
+-----------------------------------
+
+Hyperledger Grid is currently in the `incubation
+<https://wiki.hyperledger.org/display/HYP/Project+Lifecycle#ProjectLifecycle-incubation>`__
+stage of the Hyperledger product lifecycle. The `Hyperledger Grid proposal
+<https://docs.google.com/document/d/1b6ES0bKUK30E2iZizy3vjVEhPn7IvsW5buDo7nFXBE0/>`__
+was accepted in December, 2018.
+
+.. _faq-where-is-the-source-code:
+
+Where is the source code?
+-------------------------
+
+Hyperledger Grid is an open source project. The source code is in several GitHub
+repositories:
+
+* `hyperledger/grid <https://github.com/hyperledger/grid>`__: Core components
+  such as supply-chain-centric data types and smart permissioning code.
```

Comment:
Smart permissioning is a part sabre, seems weird to mention it here.

Reply:
Doesn't Pike provide a smart contract for smart permissions?  I was under the impression that Pike is in (or is moving to) the grid repo.

Reply:
Pike provides agents (with roles) and organizations that can be used in smart permissions (wasm code that returns if someone is permissioned for a task) but the actual running of smart permissions is done in Sabre Transaction Processor.

Reply:
Got it.  I'll rewrite this to mention smart contracts (instead of "smart permission code").

Reply:
Changed.